source/plugin/basics/injection.rst

```diff
@@ -0,0 +1,178 @@
+====================
+Dependency Injection
+====================
+
+Sponge uses dependency injection to provide instances of the API to plugins.
+
+Overview
+========
+
+Dependency injection allows plugins to designate a few API types that will be injected after construction.
+
+Temporary List of Injected Types
+================================
+
+* Game
```

Comment:
I'll leave that up to @BlackGoat. I didn't actually edit that part - it just looks like it because I moved the file.